metrics/scaling/k8s_scale.sh

```diff
@@ -101,6 +104,14 @@ EOF
 		cpu_idle=${cpu_idle:-0}
 		mem_free=${mem_free:-0}
 
+		# If this is the 0 node instance, store away the base memory value
+		if [ $n_pods -eq 0 ]; then
+			node_basemem[$node]=$mem_free
+		fi
+
+		local mem_used=$((node_basemem[$node]-mem_free))
+		total_mem_used=$((total_mem_used+mem_used))
```

Comment:
@dklyle - do you think I should be skipping the `noschedule` nodes here in the total mem used calculation (which is then used in the pods/gb calculation) - I suspect I should....

Reply:
yes, any of the stats should exclude the noschedule nodes. If the master node becomes a point of interest, I think that would be a separate stats table anyway.

Reply:
probably waiting on my reply, but this is not addressed in the newer commits.

Reply:
nah - I only added it a few hours ago ;-)